augur/db/export/sqlite3.py

```diff
@@ -0,0 +1,23 @@
+from augur.io.sqlite3 import Sqlite3Database
+
+
+def export(db_file:str, metadata_table:str, to_metadata:str=''):
```

Comment:
> Is that expected behavior or is the idea that this function isn't part of a public API, but it will always be called from another internal context where we know how to call it?

Yes, this is what I was thinking. In this case, the user requirement is defined on the argparse level.

We could add something like this, but it seems unnecessary:

```py
if not all(db_file, metadata_table, to_metadata):
    raise AugurError("At least one input not defined")
```

Reply:
> Will this function eventually include `to_sequences`?

Also what I was thinking.

Reply:
I don't think it needs to be an `AugurError` necessarily; an `assert` statement would do. The goal is to catch programming bugs, not user errors (because as you've said, this is an internal function).